src/AppInstallerCLITests/TestData/Manifest-Missing-MsixInstallerFields.yaml

```diff
@@ -0,0 +1,13 @@
+PackageIdentifier: AppInstallerCliTest.GoodMsixInstaller
```

Comment:
>GoodMsixInstaller [](http://example.com/codeflow?start=39&length=17)

I think this should be updated to avoid confusion. Also to keep file naming consistency, this file would be named Manifest-Bad-Missing*Fields.yaml

Reply:
Goot catch, updated and renamed